entries/n/neopets.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "NeoPass": {
+    "domain": "neopets.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://neopass.support.neopets.com/hc/articles/25933646591501",
+    "categories": [
+      "entertainment"
```

Comment:
Is it entertainment of an online game? I've heard the name Neopets before but I'm not quite sure what it is.

Reply:
The following explanation comes from [Wikipedia](https://en.wikipedia.org/wiki/Neopets):

> Neopets is a [free-to-play](https://en.wikipedia.org/wiki/Free-to-play) [virtual pet](https://en.wikipedia.org/wiki/Virtual_pet_site) [browser game](https://en.wikipedia.org/wiki/Browser_game). First launched in 1999, the game allows users to own virtual pets ("Neopets") and explore a virtual world called "Neopia." Players can earn one of two [virtual currencies](https://en.wikipedia.org/wiki/Virtual_currency). One currency, called Neopoints, can be obtained for free through on-site features like games, events, and contests. The other, Neocash (NC), is purchased with real-world money and can be exchanged for wearable items for pets.

Reply:
I think the gaming category is better.

Reply:
@hkamran80  Thanks! I missed the `gaming` category. ❤️